code/datums/diseases/advance/symptoms/heartattack.dm

```diff
@@ -0,0 +1,60 @@
+/*
+//////////////////////////////////////
+Heart Disease
+	A bit stealthy
+	Slightly resistant
+	Quite slow
+	small transmission penalty
+	max level
+Bonus
+	heart attacks
+//////////////////////////////////////
+*/
+
+/datum/symptom/heartattack
+	name = "Heart Disease"
+	desc = "This disease infiltrates the host's heart, causing cardiac arrest after a long incubation period"
+	stealth = 2
+	resistance = 1
+	stage_speed = -6
+	transmittable = -2
+	level = 8
+	severity = 6
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	var/heartattack = FALSE 
+	threshold_desc = "<b>Transmission 10:</b> When the victim has a heart attack, their heart will pop right out of their chest, and attack!.<br>\
+					  <b>Stealth 2:</b> The disease is somewhat less noticable to the host."
```

Comment:
threshold_descs = list(
		"Transmission 10" = "When the victim has a heart attack, their heart will pop right out of their chest, and attack!",
		"Stealth 2" = "The disease is somewhat less noticable to the host",
	)